pom.xml

```diff
@@ -43,8 +43,8 @@
   <dependencies>
     <dependency>
       <groupId>org.mongodb</groupId>
-      <artifactId>mongo-java-driver</artifactId>
-      <version>3.4.0</version>
+      <artifactId>mongodb-driver</artifactId>
```

Comment:
It's more common to not depend on the mongo-java-driver uber-jar, so changed the dependency to mongodb-driver.